rollup/circuitcapacitychecker/libzkp/Cargo.toml

```diff
@@ -20,8 +20,8 @@ maingate = { git = "https://github.com/scroll-tech/halo2wrong", branch = "halo2-
 halo2curves = { git = "https://github.com/scroll-tech/halo2curves.git", branch = "0.3.1-derive-serde" }
 
 [dependencies]
-prover = { git = "https://github.com/scroll-tech/scroll-prover", tag = "v0.6.4" }
-types = { git = "https://github.com/scroll-tech/scroll-prover", tag = "v0.6.4" }
+prover = { git = "https://github.com/scroll-tech/scroll-prover", tag = "v0.7.2" }
```

Comment:
Important question: This version is less strict than the previous version right? So blocks created using this version might be regarded as invalid by nodes running the old version.

Reply:
1. yes, less strict.

2. so we need to update our signer and follower nodes, and also inform our partners.
probably upgrade follower nodes, and wait for our partners to upgrade first, and then upgrade our signer?

Reply:
Then we can roll this out together with https://github.com/scroll-tech/go-ethereum/pull/476. Both require 1. update followers, 2. update signer